src/main/resources/plugin_info.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "ChatPlus",
-  "version": "1.1",
+  "version": "2.0",
```

Comment:
Lets make the version 1.2 not 2.0, its not that big of a change to require a major version change

Reply:
Done